firefox-ios/Client/Frontend/Browser/TabScrollController/TabScrollHandler.swift

```diff
@@ -172,18 +173,22 @@ final class TabScrollHandler: NSObject,
         // Ignore user scroll if the tab is loading or if the conditions to update view are not meet
         // voice over and webview's scroll content size is not enough to scroll
         guard !tabIsLoading(),
+              !isStatusBarScrollToTop,
```

Comment:
Could we have unit tests for the changes here? Is there some way to test the calculations here?